firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -273,12 +283,20 @@ private void executeFidRegistration(PersistedFidEntry persistedFidEntry, AwaitLi
     }
   }
 
-  private void updatePersistedFidWithPendingStatus(String fid) {
+  /**
+   * Refreshes FID auth token with FIS servers in a background thread and updates the listener on
+   * completion.
+   */
+  private void refreshAuthToken(AwaitListener listener) {
+    Task<InstallationTokenResult> task = Tasks.call(executor, () -> refreshAuthTokenIfNecessary(1));
```

Comment:
never pass the raw value for an `@intdef` constant; treat it like an enum and always use the predefined constants

(Gradle has a check for this afair, but I don't know if this repo is configured to run them)

Reply:
Thank you so much. Updated. Any examples on how do I configure it?